app/assets/javascripts/editor.js

```diff
@@ -23,24 +26,25 @@ $E = {
         return code;
       }
     });
-
   },
   is_editing: function() {
     return ($E.textarea[0].selectionStart == 0 && $E.textarea[0].selectionEnd == 0)
   },
-
   refresh: function() {
-      if($D.selected) {
-          $E.textarea = ($D.selected).find('textarea').eq(0);
-          $E.preview = ($D.selected).find('#preview').eq(0);
-          $E.textarea.bind('input propertychange',$E.save);
-      }
+    // textarea
+    $E.textarea = ($D.selected).find('textarea').eq(0);
+    $E.textarea.bind('input propertychange',$E.save);
+    // preview
+    $E.preview = ($D.selected).find('#preview').eq(0);
   },
```

Comment:
I didn't think these lines needed to be inside of a conditional checking for `$D.selected` (it made it little less readable). I moved the conditional into `$E.wrap`